canvas/services/event.py

```diff
@@ -52,6 +55,20 @@ def get_question_stats(question):
                 answers[answer] = 0
             answers[answer] += 1
 
+    most_recent_submission_time_for_submission_author = (
+        submissions.filter(uqj__user=OuterRef("uqj__user")).order_by("-submission_time").values("submission_time")[:1]
+    )
+
+    recent_submissions = (
+        submissions.filter(uqj__user__role="Student")
+        .annotate(recent_timestamp=Subquery(most_recent_submission_time_for_submission_author))
+        .filter(submission_time=F("recent_timestamp"))
+    )
+
+    num_students_registered_in_course = question.course.verified_course_registration.filter(
+        registration_type="STUDENT"
+    ).count()
```

Comment:
Although we are only showing this in events, the functionality works for all the questions. Even the ones that are not in a course. This code will break for the questions that are not in a course.